Fix typos to Docker 20.10 alert (#249)
home-assistant/alerts.home-assistant.io: alerts/docker_2010.markdown

On December 9, 2020 Docker released version 20.10. This version is currently not compatible with the Supervisor that is running on a Supervised installation. If you are running our Home Assistant Operating System or Home Assistant Core directly in Docker, this alert does **not** apply to you.

There are 2 parts to this issue. The first is the removal of filtering of images over the API, this results in images not being recognized by the Supervisor correctly.

The second issue is changes in how the systemd service for Docker works, which results in the service `hassio-supervisor` not being able to start on system boot.

## Workaround

> The only known workaround at this point is downgrading Docker to the previous version. The examples below are for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.

Reviewer: ```suggestion
The only known workaround at this point is downgrading Docker to the previous version. The example below is for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
```

On December 9, 2020 Docker released version 20.10. This version is currently not compatible with the Supervisor that is running on a Supervised installation. If you are running our Home Assistant Operating System or Home Assistant Core directly in Docker, this alert does **not** apply to you.

There are 2 parts to this issue. The first is the removal of filtering of images over the API, this results in images not being recognized by the Supervisor correctly.

The second issue is changes in how the systemd service for Docker works, which results in the service `hassio-supervisor` not being able to start on system boot.

## Workaround

> The only known workaround at this point is downgrading Docker to the previous version. The examples below are for Debian 10 and Ubuntu 20. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.

Reviewer: Ubuntu is not a supported system and should be removed from that.

```suggestion
The only known workaround at this point is downgrading Docker to the previous version. The examples below are for Debian 10. If that is not your OS, check the documentation for your OS's package manager on how to perform the downgrade.
```

> **Debian 10**

Reviewer: ```suggestion
```

```bash
apt install docker-ce=5:19.03.14~3-0~debian-buster
apt install docker-ce-cli=5:19.03.14~3-0~debian-buster
apt install containerd.io=1.3.9-1
```

> **Ubuntu 20**
> ```bash
> apt install docker-ce=5:19.03.14~3-0~ubuntu-focal
> apt install docker-ce-cli=5:19.03.14~3-0~ubuntu-focal
> apt install containerd.io=1.3.9-1
> ```
> 

Reviewer: Ubuntu is not supported by the Home Assistant project.
```suggestion
```

Author: Just thought there would be quite a few people looking for these commands like I was, but fair enough.
